src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -58,17 +63,20 @@ public function configure(array $configuration = null)
         parent::configure($configuration);
 
         $tagsWithNameToAlign = array_intersect($this->configuration['tags'], self::$tagsWithName);
-        $tagsWithoutNameToAlign = array_diff($this->configuration['tags'], $tagsWithNameToAlign);
+        $tagsWithMethodSignatureToAlign = array_intersect($this->configuration['tags'], self::$tagsWithMethodSignature);
+        $tagsWithoutNameToAlign = array_diff($this->configuration['tags'], $tagsWithNameToAlign, $tagsWithMethodSignatureToAlign);
 
         $indent = '(?P<indent>(?: {2}|\t)*)';
         // e.g. @param <hint> <$var>
         $tagsWithName = '(?P<tag>'.implode('|', $tagsWithNameToAlign).')\s+(?P<hint>[^$]+?)\s+(?P<var>(?:&|\.{3})?\$[^\s]+)';
         // e.g. @return <hint>
         $tagsWithoutName = '(?P<tag2>'.implode('|', $tagsWithoutNameToAlign).')\s+(?P<hint2>[^\s]+?)';
+        // e.g. @method <hint> <signature>
+        $tagsWithMethodSignature = '(?P<tag3>'.implode('|', $tagsWithMethodSignatureToAlign).')(\s+(?P<hint3>[^\s(]+)|)\s+(?P<signature>.+\))';
```

Comment:
Just a though on performance: Should this be added to the "main" regex conditionally? Like only if someone has decided to align `@method`? Would it affect performance or memory usage much?

Reply:
It would probably be better to do this only when required indeed. But I think the impact is not important and it would require to rewrite code not directly related to the new feature implemented here, so I'm fine with doing the rewrite in a separate PR. Maybe nice to know what others think though.

Reply:
Good idea, I'll make a separate PR after this is merged 😄